spec/models/rook_spec.rb

```diff
@@ -1,5 +1,45 @@
 require 'rails_helper'
 
 RSpec.describe Rook, type: :model do
-  
+  describe "valid_move? action" do
```

Comment:
Awesome! I think these are great.

It's repetitive, but I don't think we need to refactor at this point.